lib/resque-retry/server.rb

```diff
@@ -40,7 +40,7 @@ module Helpers
       # builds a retry key for the specified job.
       def retry_key_for_job(job)
         klass = get_class(job)
-        if klass.respond_to?(:redis_retry_key)
```

Comment:
With your changes to `get_class` below, just checking `klass.respond_to?(:redis_retry_key)` would be good enough here